### PR TITLE
Analogic for daterange

### PR DESCRIPTION
**Pull request by Tryxns**
This PR is to resolves Fred's request in PR https://github.com/Avaiga/taipy/pull/1991
cc: @FredLL-Avaiga

Result:
Screenshot 2024-11-05 at 12-38-20 Date Range - Example

### CODE REVIEW

File: frontend/taipy-gui/src/components/Taipy/DateRange.tsx [hunk 1]
```diff
@@ -117,6 +119,11 @@ const DateRange = (props: DateRangeProps) => {
 
     const handleChangeStart = useCallback((v: Date | null) => handleChange(v, true), [handleChange]);
     const handleChangeEnd = useCallback((v: Date | null) => handleChange(v, false), [handleChange]);
+    const analogicRenderers = {
```
Comment (FredLL-Avaiga) [inline frontend/taipy-gui/src/components/Taipy/DateRange.tsx]: outside the component ? ie before line 70

**Review by FredLL-Avaiga — CHANGES_REQUESTED** [review]

File: frontend/taipy-gui/src/components/Taipy/DateRange.tsx (hunk 1 above)
Comment (Tryxns) [reply frontend/taipy-gui/src/components/Taipy/DateRange.tsx]: Done

**Review by FredLL-Avaiga — DISMISSED** [review]
tx @Tryxns

**Comment by Tryxns** [comment]
sorry Fred, I pushed wrong commit previously (it included my local files) but now it's clean =)

**Review by FredLL-Avaiga — APPROVED** [review]

**Review by FredLL-Avaiga — CHANGES_REQUESTED** [review]
still a test failing

**Comment by Tryxns** [comment]
I think it's the same culprit `Missing 'checks: write' permission` as in https://github.com/Avaiga/taipy/pull/1991

**Comment by FredLL-Avaiga** [comment]
> I think it's the same culprit `Missing 'checks: write' permission` as in #1991

that's not what I saw

**Comment by Tryxns** [comment]
Ohh got it, fixed now

**Comment by FredLL-Avaiga** [comment]
please update your branch to get ridd of the linter issue

**Comment by FredLL-Avaiga** [comment]
I've updated your branch 
If the tests are OK, we'll merge it

**Review by FredLL-Avaiga — APPROVED** [review]

**Pull request merged**